presto-spi/src/main/java/com/facebook/presto/spi/type/AbstractLongType.java

```diff
@@ -85,6 +85,11 @@ public final void appendTo(Block block, int position, BlockBuilder blockBuilder)
         }
     }
 
+    public final void appendNonNullTo(Block block, int position, BlockBuilder blockBuilder)
+    {
+        blockBuilder.writeLong(block.getLong(position)).closeEntry();
```

Comment:
Do we know how much does this `appendNonNullTo` save?

Reply:
I don't have a benchmark for this but given that we use this for distinct and statistically nulls don't happen that often (< 5%?) in most reasonable datasets, this makes sense. I say this is my grey hair talking :)

Reply:
Removed it.